robots/cmd/kubevirtci-presubmit-creator/main.go

```diff
@@ -179,14 +179,16 @@ func AddNewPresubmitIfNotExists(jobConfig config.JobConfig, latestReleaseSemver
 
 func CreatePresubmitJobForRelease(semver *querier.SemVer) config.Presubmit {
 	yes := true
+	golangImage := "quay.io/kubevirtci/golang:v20230801-94954c0"
```

Comment:
As discussed, we should look at a way of not relying on this hardcoding but it is ok for now.